crates/sui-graphql-rpc/src/types/gas.rs

```diff
@@ -85,6 +86,7 @@ impl GasInput {
     }
 }
 
+/// The summary of the gas cost.
```

Comment:
since this is basically repeating the name of the type im not sure we need a description here

Reply:
```suggestion
/// Breakdown of gas costs in effects.
```